app/assets/javascripts/oxalis/model/actions/view_mode_actions.js

```diff
@@ -1,18 +1,76 @@
 // @flow
 
+import constants from "oxalis/constants";
 import type { OrthoViewType } from "oxalis/constants";
+import type { PartialCameraData } from "oxalis/store";
+import * as THREE from "three";
+import { getTDViewportSize } from "oxalis/model/accessors/view_mode_accessor";
 
 type SetViewportActionType = {
   type: "SET_VIEWPORT",
   viewport: OrthoViewType,
 };
 
+type SetTDCameraActionType = {
+	type: "SET_TD_CAMERA",
```

Comment:
Well spotted! I forgot to install the editor config package after I reinstalled my system. It's done now ;)